tic_tac_toe/game.py

```diff
@@ -24,17 +30,32 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    
+    # Make sure position is a tuple
+    if not type(position) == tuple:
+        return False
+    
+    # Ensure length = 2 and each entry is between 0 and 2
+    elif len(list(position)) != 2:
+        return False
+        
+    elif position[0] < 0 or position[0] > 2 or position[1] < 0 or position[1] > 2:
+        return False
+        
+    else:
+        return True
```

Comment:
Inside a function it s really to use single if statements without elif, since if it s not valid you can exit the function with return. It will make your code easier to read